.ci/jenkins_job_build.sh

```diff
@@ -72,8 +72,15 @@ fi
 # components
 .ci/setup.sh
 
-# Run the test suite
-.ci/run.sh
+# The metrics CI does not need to do the QA checks - it only runs once
+# it knows the QA CI has passed already.
+if [ -z "${METRICS_CI}" ]
+then
+	# Run the test suite
+	.ci/run.sh
+else
+	echo "Under METRICS_CI - skipping test run"
```

Comment:
generally lgtm, 
just a question, shouldn't here be a call to `run_metrics_ci.sh`, or you call it directly in your jenkins setup?

Reply:
Fine question.
I intend to run it from my Jenkins script.
Ideally I would have either:
- lifted `.ci/run.sh` out of this script all togehter
- inverted the test to only run `.ci/run.sh` if we know we are in the QA CI

but.... to do either of those would involve editing the many many Jenkins configs we have - so the easier route was to just special case the metrics.
If later we find we are having to do this again for some reason then we should consider 'biting the bullet', and updating all CIs at the same time to make this script more generically useful etc.